### ISSUE TITLE
DLCs?

### ISSUE THREAD

**Issue by darioconversano** [KuromeSan/chovy-sign#2]
Hi, is there a way that I can use my DLCs with games packed like this?

**Comment by KuromeSan** [comment]
i dont really know how PSP DLC works. xD

**Comment by LakiBR** [comment]
Yeah, you can. All you have to do is place all the files in the proper directory as like as used on PSP (specific to each game).
